infrastructure/apigee/ptl/.proxygen/credentials.yaml

```diff
@@ -0,0 +1,4 @@
+client_id: connecting-party-manager-client # must match "api" in settings.yaml
```

Comment:
Im wondering if the concept of PTL even exists in Apigee - it sounds more like we just have prod credentials, although i accept that we already put our request in with that key_id - might be worth changing it at this stage to avoid confusion?

Reply:
my impression was that there were three different sets if environments / workspaces:

* regular apim environments (ptl --> [internal-dev, internal-qa, etc], prod --> [dev, qa, int, prod], or something like that)
* proxygen environments (ptl --> [proxygen developers], prod -> [everyone else])
* cpm (dev --> [dev, local dev workspaces], ..., prod -> [prod])

and the "ptl" I've used corresponds to the regular apim environment, not the ptl one

Reply:
So are we going to need to create a second of credentials to manage the "prod" versions of regular APIM to CPM? I suppose thats actually a sensible idea even though we dont technically need two sets as a separation of concerns